docs/overview_and_guide.rst

```diff
@@ -145,6 +145,8 @@ Run the client and let it sync with the Ropsten testnet::
 
     parity --chain ropsten --bootnodes "enode://20c9ad97c081d63397d7b685a412227a40e23c8bdc6688c6f37e97cfbc22d2b4d1db1510d8f61e6a8866ad7f0e17c02b14182d37ea7c3c8b9c2683aeb6b733a1@52.169.14.227:30303,enode://6ce05930c72abc632c58e2e4324f7c7ea478cec0ed4fa2528982cf34483094e9cbc9216e7aa349691242576d552a2a56aaeae426c5303ded677ce455ba1acd9d@13.84.180.240:30303"
 
+.. attention:: Parity sometimes loses its historical DB (potentially after updated). Due to this some events might be lost which will result in Raiden not being able to fetch all event. Therefore it is recommended to make sure to have Parity fulled synced with `--no-warp`.
```

Comment:
s/potentially after updated/potentially after updates
s/able to fetch all event/able to fetch all events
s/with `--no-warp`/with the `--no-warp` option